app/screens/initiative-screen/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { observer } from 'mobx-react-lite'
+import { ViewStyle } from 'react-native'
+import { Screen, Text } from '../../components'
+// import { useNavigation } from '@react-navigation/native'
+// import { useStores } from '../../models'
+import { color } from '../../theme'
+import createScreen from '../createScreen'
+
+const ROOT: ViewStyle = {
```

Comment:
@JonnoKiwi do you want the Styling here? The template doesn't have it https://github.com/DelveFore/ignite-hydrogen/blob/master/templates/screen.ejs

Reply:
I'm finding when I run the screen generator again, the screen is created with white background and white title. So some kind of styling is needed to make the background black. 

So I could make another PR that applies a black background to the screen template. Later on I could work on connecting themes to the screen template.

Reply:
@JonnoKiwi seems like we'd need to modify the `Screen` component defaults. Let's get a GH Issue created please.

Reply:
I saw the Issue you created and then closed. I see how Hydrogen `app/theme/color` background color is already black. So I had to change it for DDIT so that the title will be visible.